lib/views.js

```diff
@@ -1,25 +1,9 @@
-exports.facilities = {
-  map: function (doc) {
-    if (doc.type === 'clinic' ||
-        doc.type === 'health_center' ||
-        doc.type === 'district_hospital' ||
-        doc.type === 'national_office' ||
-        doc.type === 'person') {
+/**
+ * This file contains all the views to be bundled up in the "medic" ddoc.
+ * If you want a view to be bundled into the "medic-client" ddoc and therefore
+ * be available on the client via pouch add it to: /ddocs/medic-client/views
+ */
```

Comment:
Understood.  Perhaps it would have been clearer to name the two ddocs `medic` and `medic-server`.